server/sonar-db-dao/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -414,6 +414,15 @@
       </if>
     </where>
   </select>
+  
+  <select id="selectUuidsByKeyFromProjectKey" parameterType="string" resultType="KeyWithUuid">
```

Comment:
FYI, using qualified name in `resultType` saves from having to declare an alias in `MyBatis.java`

Reply:
Good to know!